cosmosis/samplers/nautilus/nautilus_sampler.py

```diff
@@ -36,10 +36,9 @@ def config(self):
             self.n_live = self.read_ini("n_live", int, 2000)
             self.n_update = self.read_ini("n_update", int, self.n_live)
             self.enlarge_per_dim = self.read_ini("enlarge_per_dim", float, 1.1)
-            self.n_points_min = self.read_ini("n_points_min", int,
-                                              self.pipeline.nvaried + 50)
-            self.split_threshold = self.read_ini("split_threshold", float,
-                                                 100.0)
+            sampler_outputs = [('log_weight', float), ('prior', float),
```

Comment:
Should this be here? It looks like it might be a mistake?

Reply:
Yes, it was a mistake when I was reverting the formatting changes. It's fixed now.